app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from os import remove, rename, makedirs, path
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+    file_name = command_parts[1]
+    if "/" in command:
+        file_path, file_name = command.rsplit("/", 1)
+        makedirs(file_path)
```

Comment:
also, it makes not correct path:
1) `mv` at the created directory name
2) new file doesn't create